packages/flutter_driver/lib/src/driver/web_driver.dart

```diff
@@ -10,9 +10,13 @@ import 'package:meta/meta.dart';
 import 'package:vm_service/vm_service.dart' as vms;
 import 'package:webdriver/async_io.dart' as async_io;
 import 'package:webdriver/support/async.dart';
+import 'package:file/file.dart' as f;
```

Comment:
We generally don't import file with a prefix.

Reply:
fixed.